gruntfile.js

```diff
@@ -204,6 +209,33 @@ function shell_download_docker_binary(p, a) {
   }
 }
 
+function shell_download_docker_compose_binary(p, a) {
+  var ps = { windows: 'win', darwin: 'mac' };
+  var as = { amd64: 'x86_64', arm: 'armhf', arm64: 'aarch64' };
+  var ip = ps[p] === undefined ? p : ps[p];
+  var ia = as[a] === undefined ? a : as[a];
+  var binaryVersion = '<%= binaries.dockerComposeVersion %>';
+
+  if (p === 'linux' || p === 'mac') {
+    return [
```

Comment:
does our script handle `mac`? it also doesn't handle anything other than `amd64` so maybe just return an error for these cases? or an empty response?

Reply:
thanks I will update them in https://github.com/portainer/portainer/pull/4713